pkgs/applications/misc/calibre/default.nix

```diff
@@ -99,7 +99,7 @@ mkDerivation rec {
       sip
       # the following are distributed with calibre, but we use upstream instead
       odfpy
-    ]
+    ] ++ lib.optional (unrarSupport) unrardll
```

Comment:
This PR won't fix the issue if unrarSupport is `false` by default.

Reply:
I am very much against making calibre unfree by default, so unrarSupport should stay false.

Reply:
Agree. Forgot that rar is unfree.